spec.md

```diff
@@ -115,6 +116,11 @@ Return a list of counties in `Division` data structure.
 A `prefecture_code` is a 6-length code that endswith `00`. It can also be a 4-length code.
 
 
+### `#revisions`
```

Comment:
@lepture It reminds me. I agree with "even a constant should be implemented as a callable method". It is similar with `getter method` in Java / C. A method always can be refactored without compatible changes, and keeps memory-layout independent in static languages.